### PR TITLE
Add --set-default flag.

### PR DESCRIPTION
**Pull request by rbartlensky**
This PR adds a new `--set-default` flag which enables users to set a default username for a service. It also removes the `username` parameters of `get` and `set`. Instead, the `pwvlt` crate will look for a default username, and if one is not found, the user is prompted for it.

### CODE REVIEW

File: pwvlt/src/util.rs [hunk 1]
```diff
@@ -42,3 +42,12 @@ where
         }
     }
 }
+
+pub fn prompt_string<S: AsRef<str>>(message: S) -> String {
```
Comment (gabi-250) [inline pwvlt/src/util.rs:41]: You can refactor `looping_prompt` to use `prompt_string`.

File: pwvlt-cli/src/config.rs [hunk 2]
```diff
@@ -10,12 +10,14 @@ const DEFAULT_CONFIG: &str = "# Autogenerated pwvlt config
...
 [password]
-length = 22
+length = 20
 numbers = true
 lowercase_letters = true
 uppercase_letters = true
 symbols = true
 strict = true
+
+[default]
```
Comment (gabi-250) [inline pwvlt-cli/src/config.rs]: I think this should be optional.

**Review by gabi-250 — CHANGES_REQUESTED** [review]

**Comment by rbartlensky** [comment]
@gabi-250 all done, ready for another review.

File: pwvlt-cli/src/main.rs [hunk 3]
```diff
@@ -22,29 +22,35 @@ fn main() {
...
+                .args(&["set", "get", "set-default"]),
         )
         .get_matches();
     let pv = PasswordVault::new(config::load_config().unwrap());
     if let Some(mut values) = matches.values_of("get") {
         let service = values.next().unwrap();
-        let username = values.next().unwrap();
+        let username = match pv.default(&service) {
+            Some(username) => username.to_string(),
+            None => prompt_string(format!("Enter username for {}", service)),
```
Comment (gabi-250) [inline pwvlt-cli/src/main.rs:52]: `prompt_string` should probably `trim` the string it returns.

Comment (rbartlensky) [reply pwvlt-cli/src/main.rs:52]: nice one, completely missed that!

**Review by gabi-250 — APPROVED** [review]

**Comment by gabi-250** [comment]
You can squash the last 3 commits if you want.

**Comment by rbartlensky** [comment]
@gabi-250 done!

**Pull request merged**